tests/assertions.js

```diff
@@ -20,20 +20,38 @@ export function shouldReturnResult({result, error}) {
   should.exist(result, 'Expected a result');
 }
 
-export function testIssuedVc({issuedVc}) {
-  issuedVc.should.be.an('object');
+export function shouldBeIssuedVc({issuedVc}) {
+  issuedVc.should.be.an(
+    'object',
+    'Expected the issued Vc to be an object.'
+  );
   issuedVc.should.have.property('@context');
   issuedVc.should.have.property('type');
-  issuedVc.type.should.contain('VerifiableCredential');
+  issuedVc.type.should.contain(
+    'VerifiableCredential',
+    'Expected Vc type to contain "VerifiableCredential."'
+  );
   issuedVc.should.have.property('id');
-  issuedVc.id.should.be.a('string');
+  issuedVc.id.should.be.a(
+    'string',
+    'Expected Vc id to be a string.'
+  );
   issuedVc.should.have.property('credentialSubject');
-  issuedVc.credentialSubject.should.be.an('object');
+  issuedVc.credentialSubject.should.be.an(
+    'object',
```

Comment:
https://www.w3.org/TR/vc-data-model/#credential-subject - credentialSubject should be oneOf - array or object

Reply:
This assertion is actually kind of hard to do:

1. [`oneOf` is not an assertion on type actually.](https://www.chaijs.com/api/bdd/#method_oneof)
2. `typeof` returns the primitive type, but `Array` is not a primitive type it's actually an object
3. `typeof []` is object
4. `typeof {}` is object
5. So the existing assertion should work, but chai is a bit to smart so [].should.be.an('object') does fail

What we really need is a compound assertion:

`credentialSubject.should.be.an('object').or.an('Array')` I guess what I'm trying to get at is that it's not as easy as `const csType = typeof vc.credentialSubject` and then `csType.should.be.oneOf(['string', 'array'])`

Reply:
Issue added here: https://github.com/w3c-ccg/vc-api-issuer-test-suite/issues/26